src/vcpkg/vcpkgpaths.cpp

```diff
@@ -1011,38 +1013,41 @@ namespace vcpkg
         auto maybe_tar_output = flatten(cmd_execute_and_capture_output(tar_cmd_builder), Tools::TAR);
         if (!maybe_tar_output)
         {
-            auto message =
-                Strings::concat(PRELUDE, "Error: Failed to tar port directory\n", std::move(maybe_tar_output).error());
-
+            auto message = msg::format_error(msgGitCommandFailed, msg::command_line = tar_cmd_builder.command_line());
```

Comment:
This no longer includes `maybe_tar_output` in the output.

Reply:
It's now added on 1027.